syft/pkg/cataloger/r/package.go

```diff
@@ -0,0 +1,38 @@
+package r
+
+import (
+	"github.com/anchore/packageurl-go"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/source"
+)
+
+func newPackage(pd parseData, locations ...source.Location) pkg.Package {
+	locationSet := source.NewLocationSet()
+	for _, loc := range locations {
+		locationSet.Add(loc.WithAnnotation(pkg.EvidenceAnnotationKey, pkg.PrimaryEvidenceAnnotation))
+	}
+	result := pkg.Package{
+		Name:         pd.Package,
+		Version:      pd.Version,
+		FoundBy:      catalogerName,
```

Comment:
fun fact: the found-by field isn't set by the cataloger function itself, but instead by the caller. Specifically the generic cataloger object does this https://github.com/anchore/syft/blob/main/syft/pkg/cataloger/generic/cataloger.go#L137 (which yours is using)

Reply:
Should I remove this then?

Reply:
Will remove.